test/test_issue655.py

```diff
@@ -0,0 +1,27 @@
+from rdflib import Graph, Namespace, URIRef, Literal
+from rdflib.compare import to_isomorphic
+import unittest
+
+
+class TestIssue655(unittest.TestCase):
```

Comment:
could you add a simple additional test for:
- `Literal(float("inf")).n3 == '"INF"^^xsd:float'` 
- `Literal(float("-inf")).n3 == '"-INF"^^xsd:float'` 

Also, doesn't the same problem occur for NaN?